specs/place-service.spec.php

```diff
@@ -58,6 +67,14 @@
             $this->prophet->checkPredictions();
         });
 
+        it('should encode optional params', function () {
+            $this->client->fetch('https://maps.googleapis.com/maps/api/place/findplacefromtext/json?key=master&input=foo&inputtype=textquery&bar=baz+biz')
+                ->willReturn(['candidates' => 9])->shouldBeCalled();
+            $this->service->findPlace('foo', null, null, ['bar' => 'baz biz']);
+
+            $this->prophet->checkPredictions();
```

Comment:
Should checkPredictions go in an 'afterEach' block?

Reply:
Sure. That's a convention we've used at VNN for a less verbose test file. Most tests here had the check at the end of each individual test, so I just stuck with convention. The next time we need to change this library, we might as well update the test file, too.